### ISSUE TITLE
X-ray appears randomly with Better Foliage enabled

### ISSUE THREAD

**Issue by Artwind2** [the-realest-stu/Dynamic-Trees-BOP#9]
I've got some X-rays around Dynamic Trees, more frequently on big forests.
It stops when Better Foliage is disabled, and I didn't have this issue in a vanilla world.

![2018-04-12_19 06 25](https://user-images.githubusercontent.com/38019992/38697809-5290df18-3e93-11e8-88cb-8e790b36c289.png)
![2018-04-12_19 31 33](https://user-images.githubusercontent.com/38019992/38697820-57b01270-3e93-11e8-92b9-63e64d645c48.png)

**Comment by ferreusveritas** [comment]
Whoa.  Haven't seen this behavior before.  Even with optifine and shaders.  I wonder if there is a limit to how many quads a chunk can be baked with.  I'd hate to pin it on better foliage without more investigation.  Does this by chance happen with better foliage installed and not dynamic trees?

**Comment by Artwind2** [comment]
So, I made some other tests :
-Dynamic Trees OFF -Better Foliage ON : same issue but rare. One time when looking a cliff. And experienced also on a Recurrent Complex Village (and I remember having this on almost every village I visited before)
-Dynamic Trees ON - Better Foliage ON : issue appears (like a mentioned first)
-Dynamic Trees OFF -Better Foliage OFF : fine
-Dynamic Trees ON - Better Foliage OFF : fine

Your mod doesn't seems responsible of this issue. I'll investigate on Better Foliage side

By the way I'm using Sildurs shader with optifine.

Thank you for you quick answer, and for your very fancy mod (can't wait to see larger trees!)
